setup.py

```diff
@@ -42,7 +42,11 @@
 
 BASE_PATH = os.path.dirname(os.path.abspath(__file__))
 SRC_PATH = os.path.join(BASE_PATH, "src")
-PACKAGES = find_packages(where=SRC_PATH)
+PACKAGES = find_packages(where=SRC_PATH,
```

Comment:
Complicated question ... I first moved the one ... create_schedule or so function into utilities, before I noticed the side effects (tests were installed etc), at which stage I created a separate branch just for that and created #511, then moved this to be based on #511 (since it already had code to move that function). A bit messy, but most of the work will be in merging, which I will handle - that's what I meant with "I had some merge conflicts", basically changes to the function that I moved into psyclone_utilities (which has now been renamed *sigh* merging will be a pain). Still, we should get #511 in first.

Reply:
OK :-)